defaults/main.yml

```diff
@@ -145,6 +145,9 @@ grafana_smtp: {}
 # Enable grafana alerting mechanism
 grafana_alerting: true
 
+# Execute grafana alerts
+grafana_execute_alerts: true
```

Comment:
This is perfect, but could you change variable name to `grafana_alerting_execute_alerts`? This way it will be known that this belongs to `alerting` section.

Other solution would be to make `grafana_alerting` a map and apply mechanisms we are also using for `grafana_metrics`. This way we would have a consistent way of setting those variables.

Reply:
I thought about making it a map, it looks to me that we might need to add extra logic to handle backwards compatibility.

Reply:
I've renamed the variable and simplified the logic (no if/else).

Reply:
If we make it a map now, then backwards compatibility will be simple to do as empty dict maps to "false" and a dict with anything is "true".

However if I merge this now, then backwards compatibility for a map will be much more complex.